src/status_im/utils/security.cljs

```diff
@@ -17,3 +18,8 @@
 ;; Returns a MaskedData instance that stores the piece of data.
 (defn mask-data [data]
   (MaskedData. data))
+
+(defn unmask-data [data]
```

Comment:
looks unused. either we need to use `unmask-data` in the code, or remove this method

Reply:
https://github.com/status-im/status-react/pull/6431/files#diff-b2c790594a17c6e0263a23b429552b16R103

Reply:
ah, okay